util/convert_wycheproof/convert_wycheproof.go

```diff
@@ -26,11 +26,11 @@ import (
 )
 
 type wycheproofTest struct {
-	Algorithm        string            `json:"algorithm"`
-	GeneratorVersion string            `json:"generatorVersion"`
-	NumberOfTests    int               `json:"numberOfTests"`
-	Notes            map[string]string `json:"notes"`
-	Header           []string          `json:"header"`
+	Algorithm        string         `json:"algorithm"`
+	GeneratorVersion string         `json:"generatorVersion"`
+	NumberOfTests    int            `json:"numberOfTests"`
+	Notes            map[string]any `json:"notes"`
```

Comment:
just curious, why the change from `string` to `any`?

Reply:
I'm not sure why, but the new test vectors have a `notes` section that's a struct rather than a string.